internal/common/database/migrations_test.go

```diff
@@ -0,0 +1,77 @@
+package database
+
+import (
+	"testing"
```

Comment:
Can you run the `goimports` tool on this file? It will sort the imports, and separate out third-party/non-standard-library modules, automatically.

Reply:
Okay let me do it